### ISSUE TITLE
[FEATURE] Move Position of search bar

### ISSUE THREAD

**Issue by Dun-sin** [Njong392/Abbreve#89]
### Description

I think it's much better if the search bar is located at the top, it's easier to find.
Just a thought

**Comment by adropofilm** [comment]
I agree it should probably be moved somewhere that gives it more attention, because right now the end user has to look for it more than they probably should. The search component blends in in the same way an email subscription input does. I think it might look nice underneath the abreviation examples, as highlighted in the image below. 

![Screen Shot 2022-10-06 at 2 10 01 AM](https://user-images.githubusercontent.com/9134674/194238371-9d581027-884a-405a-9bb4-9674895bbd7b.jpg)

Thoughts @Dun-sin @Njong392 ?

**Comment by Dun-sin** [comment]
> I agree it should probably be moved somewhere that gives it more attention, because right now the end user has to look for it more than they probably should. The search component blends in in the same way an email subscription input does. I think it might look nice underneath the abreviation examples, as highlighted in the image below. 
> 
> ![Screen Shot 2022-10-06 at 2 10 01 AM](https://user-images.githubusercontent.com/9134674/194238371-9d581027-884a-405a-9bb4-9674895bbd7b.jpg)
> 
> Thoughts @Dun-sin @Njong392 ?

I was thinking either there or the navbar section
